modular-docs-manual/content/topics/module_modular-documentation-repositories.adoc

```diff
@@ -0,0 +1,6 @@
+[id="modular-documentation-repositories"]
+= Modular Documentation Repositories
+
+Due to many factors that determine documentation repository design for individual products, the modular documentation initiative cannot provide a strict documentation repository template for all documentation teams to follow. Each team faces different implementation challenges, especially when Red Hat enterprise product documentation is single-sourced with corresponding community project documentation.
+
+However, for a general repository structure that the modular documentation initiative recommends for optimal results with modular standards, see the example documentation repository and `README` file in the https://github.com/redhat-documentation/modular-docs/tree/mod-doc-repo-example[`mod-doc-repo-example`] branch in this repository.
```

Comment:
```suggestion
For a general repository structure used by documentation teams at Red Hat, see the example documentation repository and `README` file in the https://github.com/redhat-documentation/modular-docs/tree/mod-doc-repo-example[`mod-doc-repo-example`] branch in this repository.
```

Reply:
Again, wanting to be clear, but also, this rewording is necessarily correct. It's not a repo design "used by doc teams at Red Hat", at least not yet. Only a few teams are somewhat similar to this. Otherwise, teams are all over the map. That's kind of the purpose of the repo example, to give a recommended structure for optimal results with mod doc. So I guess I again would prefer the existing wording over the rewording.

Reply:
* I want to make it clear that the example is RH-specific -- as discussed. We can qualify the statement ("some teams...").
* I I think it's superfluous to include any wording referring to a "modular docs initiative". This entire manual is a product of some initiative. In product docs, we don't say "The Customer Content Services team recommends..." :)
```suggestion
For a general repository structure used by some documentation teams at Red Hat, see the example documentation repository and `README` file in the https://github.com/redhat-documentation/modular-docs/tree/mod-doc-repo-example[`mod-doc-repo-example`] branch in this repository.
```